src/git.ts

```diff
@@ -42,9 +42,8 @@ export async function init(): Promise<void> {
     await execute(`git init`, workspace);
     await execute(`git config user.name ${action.name}`, workspace);
     await execute(`git config user.email ${action.email}`, workspace);
-    await execute(`git remote rm origin`, workspace);
```

Comment:
Is `git remote rm deployment-repo` not applicable here?

Reply:
I don't believe so, as the fresh Action execution context won't have a concept of non-origin remotes each time. I could be mistaken (only started working with Actions this weekend).